packages/EasyCodingStandard/src/Application/Application.php

```diff
@@ -100,6 +100,10 @@ public function run(): int
             $this->processFoundFilesSecondRun($files);
         }
 
+        if ($this->configuration->showProgressBar() && count($files)) {
+            $this->easyCodingStandardStyle->progressFinish();
+        }
```

Comment:
This adds 2 empty extra lines to the output:

![image](https://user-images.githubusercontent.com/924196/49329744-32b4d480-f584-11e8-8173-d6c49409b13e.png)

What is it good for?

Reply:
Screenshot is enough :)

Testing console output is like writing mock for every method

Reply:
Nice article. I will have to start reading your blog more. Lots of experience there.